app.ts

```diff
@@ -0,0 +1,39 @@
+import { Request, Response } from "express-serve-static-core";
+const session = require('express-session');
+
+const express = require("express");
+const path = require("path");
+const nunjucks = require("nunjucks");
```

Comment:
For imports the typescript way is to use import instead of require. For example:

```
import expressSession from 'express-session';
import express, { Express } from 'express';
import nunjucks from 'nunjucks';
import path from 'path';
```

Reply:
The stuff in the academy uses commonjs rather than esm though which is annoying

Reply:
Fixed